### PR TITLE
add JBT 4.3.1.Final to tools.jboss.org (JBDS...

### PR DESCRIPTION
**Pull request by nickboldt**
add JBT 4.3.1.Final & JBDS 9.1.0 to tools.jboss.org

### CODE REVIEW

File: _config/products.yml [hunk 1]
```diff
@@ -74,10 +74,41 @@ devstudio:
   url_path_fragment: devstudio
   streams:
     mars:
+      9.1.0.GA:
+        update_site_url: https://devstudio.redhat.com/9.0/stable/updates/
+        marketplace_url: https://marketplace.eclipse.org/node/1616973
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2016-04-19
+        # TODO change this to 9.1.0.GA when they release their bits
+        supported_devstudio_is_version: 9.0.0.Beta1
```
Comment (alexeykazakov) [inline _config/products.yml]: Does this supported_devstudio_is_version mean anything? Or just redundant?

File: _config/products.yml [hunk 2]
```diff
@@ -74,10 +74,41 @@ devstudio:
...
+            file_size: 505MB
+          - name: Installer With EAP 6.4.0 CVE-2015-7501
+            url: http://www.jboss.org/download-manager/content/origin/files/sha256/39/391763fcfcaa21906cc4d1adbf0700fd21ece652bca4d7a3985b56e414270ff5/jboss-devstudio-9.1.0.GA-installer-eap.jar
+            file_size: 651MB
+        zips:
+          - name: Update site (including sources) bundle of JBoss Developer Studio
+            url: http://www.jboss.org/download-manager/content/origin/files/sha256/b6/b6541031bc3b3a8ffc5cced966e7ceda99817bbe9e3423741afa83da44d83d0f/jboss-devstudio-9.1.0.GA-updatesite-core.zip
+            file_size: 528MB
+          - name: Target Platform for JBoss Developer Studio
+            url: https://devstudio.redhat.com/9.0/development/updates/core/jboss-devstudio-9.1.0.GA-target-platform.zip
```
Comment (alexeykazakov) [inline _config/products.yml]: The TP is not available.

File: _config/products.yml (hunk 1 above)
Comment (nickboldt) [inline _config/products.yml]: Ask Paul. Pretty sure that YES, it does.

File: _config/products.yml (hunk 2 above)
Comment (nickboldt) [inline _config/products.yml]: Fixed; had wrong path. Now using /static/ for better Akamai performance.

File: _config/products.yml (hunk 1 above)
Comment (alexeykazakov) [inline _config/products.yml]: Ok. Then what does this mean?

**Comment by alexeykazakov** [comment]
Rebased and pushed to master and production.

File: _config/products.yml (hunk 1 above)
Comment (nickboldt) [inline _config/products.yml]: As I'm sure @pleacu will tell you, that's the latest current version of JBDS IS available.

If you have questions about the yaml logic, I'm sure Xavier can explain it.

Comment (alexeykazakov) [inline _config/products.yml]: Ah, I completely missed that this is about JBDS IS ;)

**Pull request closed without merging**